numpy/__init__.pxd

```diff
@@ -524,8 +517,8 @@ cdef extern from "numpy/arrayobject.h":
     # more than is probably needed until it can be checked further.
     int PyArray_SetNumericOps        (object)
     object PyArray_GetNumericOps ()
-    int PyArray_INCREF (ndarray)
-    int PyArray_XDECREF (ndarray)
+    # int PyArray_INCREF (ndarray)
+    # int PyArray_XDECREF (ndarray)
```

Comment:
I'm curious about the removal of these two functions, and the two below. They are part of the API, does cython not want to use them?

Reply:
Correct, thanks. Reverting this change. @scoder: fyi